Assets/Scripts/LevelEditor/LevelEditor.cs

```diff
@@ -166,19 +214,39 @@ private void LoadLevel()
 
     private async void Publish()
     {
+        publishButton.interactable = false;
+
+        GameObject.Find("Error Messages").GetComponent<Text>().text = "";

```

Comment:
AHHHH NO. No. Bad. NEVER use this function, it's the most inefficient code possible lmao